front_end/panels/ai_chat/ui/AIChatPanel.ts

```
@@ -718,6 +722,7 @@ export class AIChatPanel extends UI.Panel.Panel {
   #liteLLMEndpoint: string | null = null; // LiteLLM endpoint
   #apiKey: string | null = null; // Regular API key
   #evaluationAgent: EvaluationAgent | null = null; // Evaluation agent for this tab
+  #mcpUnsubscribe: (() => void) | null = null;
```

ESLint

[error] '#mcpUnsubscribe' is defined but never used.

(no-unused-private-class-members)